spec/stringProcessing/indicesSpec.js

```diff
@@ -92,6 +93,20 @@ describe( "A function to filter out overlapping matches", function() {
 
 });
 
+describe( "A function that returns all instances of the input word and their index", function() {
```

Comment:
Another comment on the tests file in general: How do I know what function is tested by a test specifically, since only a description is given?